forms-flow-web/src/components/Form/EditForm/FormSettings.js

```diff
@@ -89,6 +93,36 @@ const FormSettings = forwardRef((props, ref) => {
 
   });
 
+  /* --------Updating path if multitenant enabled-------------------------- */
+  useEffect(()=>{
+    if(MULTITENANCY_ENABLED){
+      const updatedDisplayPath = removeTenantKeyFromPath(formDetails.path,tenantKey);
+      setFormDetails((prev) => {
+        return {
+          ...prev,
+          path: updatedDisplayPath
+        };
+      });
+      const updatedUrlPath = addTenantkeyAsSuffix(publicUrlPath,tenantKey);
+      setUrlPath(updatedUrlPath);
+    }
+  },[MULTITENANCY_ENABLED]);
+
+  const addTenantkeyAsSuffix = (value, tenantkey) => {
+    if (value.toLowerCase().endsWith(`-${tenantkey}`)) {
+      return value.toLowerCase();
+    } else {
+      return `${value.toLowerCase()}${tenantkey}-`;
+    }
+  };
+
+  const removeTenantKeyFromPath = (value, tenantKey) => {
+    const tenantKeyCheck = value.match(`${tenantKey}-`)?.[0];
+    return tenantKeyCheck?.toLowerCase() === `${tenantKey.toLowerCase()}-`
+      ? value.replace(`${tenantKey.toLowerCase()}-`, "")
+      : value;
+  };
```

Comment:
can we handle this in a helper function so that this can be reused later
? 

@fahad-aot

Reply:
Sure ,Moved to helper.